src/App.tsx

```diff
@@ -1,34 +1,53 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { getTodos } from './api';
+import { actions as todoAction } from './features/todos';
 
 export const App: React.FC = () => {
+  const [loader, setLoader] = useState(true);
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const dispatch = useAppDispatch();
+
+  const loadTodos = useCallback(async () => {
+    try {
+      const loadedTodos = await getTodos();
+
+      dispatch(todoAction.setTodos(loadedTodos));
+      setLoader(false);
+    } catch {
+      // handleErrorSet(ErrMessage.loadTodo);
+    }
+  }, []);
```

Comment:
your dependency arrays are missing some values. pls fix this
![image](https://github.com/mate-academy/react_redux-list-of-todos/assets/108538921/58222815-48fa-4905-b77f-6cc62c0b2c09)